src/components/SideBar.tsx

```diff
@@ -1,78 +1,262 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
+import BrainBeatsAudioPlayer from '@brainbeatsucf/brainbeats-audio-player';
+import '@brainbeatsucf/brainbeats-audio-player/src/style.css';
+import { Link } from 'react-router-dom';
+import { useAuth0 } from '@auth0/auth0-react';
+
+interface SideBarProps {
+  isPlaying: boolean,
+  togglePlayPauseButon: any,
+  setAudioGlobal: any,
+  id: number
+}
+
+interface AudioObject {
+  imageUrl: string,
+  audioUrl: string
+  title: string,
+  authorName: string,
+}
 
 const useStyles = makeStyles(() => ({
-  sidebar: {
-    height: '100%',
-    color: 'white',
+  fixedLeftBottom: {
+    position: 'fixed',
+    bottom: 0,
+    width: '100%',
     backgroundColor: '#1a1919',
   },
```

Comment:
Can these `useStyles` object be put in their own files? They seem big enough to warrant that